gcloud/connection.py

```diff
@@ -219,6 +225,10 @@ def _do_request(self, method, url, headers, data):
         :type data: string
         :param data: The data to send as the body of the request.
 
+        :type dummy: object or :class:`NoneType`
+        :param dummy: Unused ``target_object`` here but may be used
+                      by a superclass.
```

Comment:
I agree about what Pylint ought to do, but that's not what it does do.

We could either do an inline `pylint: disable` or add the variable name to

```
dummy-variables-rgx=_$|dummy
```

in `pylintrc_default`.

WDYT?